packages/server/src/webhooks/broadcaster.ts

```diff
@@ -32,6 +38,8 @@ export class WebhookBroadcaster {
     }
 
     try {
+      this.logger.debug(`Client ID '${clientId}' calling webhook URL '${url}'`)
```

Comment:
You need to check if logging is enabled.

```ts
if (yn(process.env.LOGGING_ENABLED)) {
  this.logger.debug(`Client ID '${clientId}' calling webhook URL '${url}'`)
}
```

Reply:
It would be better if this logic was inside the logger itself so we don't have to make that check each time we call debug!

Reply:
We'll do something similar to that later. For now it's necessary to check if logging is enabled that way otherwise botpress will be bombarded with logs